lnclient/ldk/ldk.go

```diff
@@ -1772,3 +1786,18 @@ func forceCloseChannelsFromStaticChannelsBackup(node *ldk_node.Node, staticChann
 
 	node.ForceCloseAllChannelsWithoutBroadcastingTxn()
 }
+
+func GetVssNodeIdentifier(keys keys.Keys) (string, error) {
+	key, err := keys.DeriveKey([]uint32{bip32.FirstHardenedChild + 2})
+
+	if err != nil {
+		return "", err
+	}
+
+	pubkeyBytes, err := key.PublicKey().Serialize()
+	if err != nil {
+		return "", err
+	}
+
+	return hex.EncodeToString(pubkeyBytes[0:3]), nil
```

Comment:
why such a short identifier? and not the whole?

Reply:
It's a 6-digit hex number e.g. `0488b2`, which should be a one in over a million chance of having a collision if the same user runs with a different seed. I am not sure there is any need to include the whole derived pubkey on this specific path and makes the identifier very long. What do you think? It is only used to ensure the user can run multiple nodes with the same alby account.